serialize/src/lib.rs

```diff
@@ -1,25 +1,65 @@
 use bytes::Buf;
 use log::warn;
 use sha2::{Digest, Sha256};
+use std::convert::Infallible;
+use std::fmt::{Display, Formatter};
 use std::io::{Cursor, Error, ErrorKind, Read};
+use std::str::FromStr;
 
 pub fn hash_256(input: impl AsRef<[u8]>) -> Vec<u8> {
     let mut hasher = Sha256::new();
     hasher.update(input);
     hasher.finalize().to_vec()
 }
 
+#[derive(Default, Copy, Clone, Ord, PartialOrd, Eq, PartialEq)]
+pub enum ChiaProtocolVersion {
+    Chia0_0_34 = 34, //Pre 2.0.0
+    #[default]
```

Comment:
Does this mean the default is 35 instead of 36?

Reply:
It was while the 2.2.0 changes were pending. With the 2.2.1 release I have switched this back